src/manual/coupon-details.md

```diff
@@ -9,58 +9,71 @@ Label (required)
+
+!!! hint
+    Up to and including *Aimeos* 20.10.2, uninstalling a decorator has to be done manually by drag-selecting the decorator name including its preceding comma, hitting the delete key or cutting out the selected text and saving afterwards.  
+    Starting with *Aimeos* 20.10.3, this field is read only. The only way to delete a decorator now is by replacing the current configuration completely via adding the provider anew and re-assigning decorators again.
+    
+    A decorator's configuration field(s) will not be deleted right away, which might be helpful for testing purposes. If you don't know, which configuration fields do not belong to the current setup anymore, simply hit *Save* in the top right corner of the page: the page will refuse to save and instead warn about all superfluous fields.
 
 # Coupon configuration
 
-The right side (or below) of the coupon detail view contains a key/value configuration panel where the necessary configuration values of the coupon and the decorators must be added. The left column in the configuration panel is for the configuration key, the right column for the value and at least the left column of each line must be filled with a valid key. You can add lines by clicking on the "Add" button in the menu bar and delete selected lines with the "Delete" button. The order of the lines doesn't matter.
+The right side (or lower part) of the coupon detail view offers a panel, which presents a list of all the configurable options of a coupon provider and its assigned decorators. Whenever you add or remove a provider or a decorator, its respective configuration option(s) will be added to or removed from the list automatically.
+
+The panel's left column is resevered for configuration keys, the right column for their values. At least the left column of each line must be filled with a valid key. You can add lines by clicking on the "Add" (+) button in the menu bar and delete selected lines via the respective "Delete" (x) button on a line's right side. The order of the lines doesn't matter.
 
 ![Coupon configuration](Admin-backend-coupon-config.png)
 
-Each coupon and decorator needs its own configuration. For details about the required resp. available configuration keys and values, please refer to the documentation of the coupon or decorator. Documentation for the built-in ones is available at the [coupon overview](coupons.md) page and the [coupon decorators](coupon-decorators.md) page. When you add a provider or a decorator, their configuration options will be added to the list automatically.
+Each coupon and each decorator needs its own configuration. For details about which configuration keys and values are available resp. required, refer to the documentation of the coupon or decorator. Documentation for *Aimeos*'s built-in coupons is available in [Available coupons](coupons.md) and [Adding features](coupon-decorators.md).
```

Comment:
Maybe add the word "article" or "section" at the end and lowercase "Available" and "Adding"

Reply:
The last sentence needs re-phrasing, true, but adding "article" and lowercasing the title is wrong. Referring to a title or a chapter always has to be capitalised. (To be totally correct, "Features" in "Available features" must be capitalised, too (only words like "the" or "in" may remain lowercase in titles); but that would mean to change the whole documentation.)
As for "adding article": you wouldn't add "article", when you link to wikipedia page, either, would you? The link is signal enough that  a new section/article/chapter/document/page/site will be opened.